pr_agent/servers/bitbucket_server_webhook.py

```diff
@@ -13,35 +14,55 @@
 from pr_agent.agent.pr_agent import PRAgent
 from pr_agent.config_loader import get_settings
 from pr_agent.log import get_logger
+from pr_agent.servers.utils import verify_signature
 
 router = APIRouter()
 
 
-def handle_request(background_tasks: BackgroundTasks, url: str, body: str, log_context: dict):
+def handle_request(
+    background_tasks: BackgroundTasks, url: str, body: str, log_context: dict
+):
     log_context["action"] = body
-    log_context["event"] = "pull_request" if body == "review" else "comment"
     log_context["api_url"] = url
     with get_logger().contextualize(**log_context):
         background_tasks.add_task(PRAgent().handle_request, url, body)
 
 
-@router.post("/webhook")
+@router.post("/")
```

Comment:
Why did you change the webhook endpoint to /?

Reply:
Because on the configuration page in bitbucket, there is a Test connection button, if the webhook has the same get and post URL you can see if the call succeeds or not otherwise this test will always fail.

![image](https://github.com/Codium-ai/pr-agent/assets/28878/7a1a65f9-7210-46e9-8f67-8e604b2f8cca)